src/test/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceServiceTest.groovy

```diff
@@ -254,18 +254,38 @@ class SurrealPersistenceServiceTest extends Specification implements SurrealDBTe
         def addr = "100.200.300.400"
         def exp = Instant.now().plusSeconds(3600)
         and:
-        def request = new WaveContainerRecord(req, data, wave, addr, exp)
+        def request = new WaveContainerRecord(req, data, TOKEN, wave, addr, exp)
 
         and:
-        persistence.saveContainerRequest(TOKEN, request)
+        persistence.saveContainerRequest(request)
         and:
         sleep 200  // <-- the above request is async, give time to save it
         
         when:
         def loaded = persistence.loadContainerRequest(TOKEN)
-        then:
-        loaded == request
 
+        then:
+        verifyAll(loaded) {
+            loaded.id == "wave_request:$request.id" //surrealdb will add table name in front of id
```

Comment:
It would be better to avoid this. Maybe a getter stripping the prefix can do the trick, see https://github.com/seqeralabs/wave/blob/5589965bd1f4d1b4403ec1be42f4282a57af17ba/src/main/groovy/io/seqera/wave/util/StringUtils.groovy#L71-L72

Reply:
done